controllers/movies.js

```diff
@@ -0,0 +1,41 @@
+const movies = require('../movies')
+
+const getAllMovies = (req, res) => {
+  res.send(movies)
+}
+
+const searchByTitleAndDirector = (req, res) => {
+  const qSearch = req.params.query.toLowerCase()
+
+  const movieResults = movies.filter(movie => movie.title.toLowerCase().includes(qSearch))
+
+  const directorResults = movies.filter(movie => {
+    for (director of movie.directors) {
+      if (director.toLowerCase().includes(qSearch)) {
+        return movie
+      }
+    }
+  })
+
+  const combinedResults = movieResults.concat(directorResults)
```

Comment:
What if movie is found by title and director... you will concat and list the movie twice?